pyproject.toml

```diff
@@ -61,16 +62,16 @@ phonenumberslite = "^8.10.16"
 prices = "^1.0"
 psycopg2-binary = "^2.8.3"
 purl = "^1.5"
+python-magic-bin = "^0.4.14"
```

Comment:
This will install `python-magic-bin` on all platforms, right? Is this what we want or we want to only install it on Windows?

Reply:
You're right. I must have missed the marker.